src/Orchestration/Adapter/DockerCLI.php

```diff
@@ -102,6 +102,135 @@ public function networkDisconnect(string $container, string $network, bool $forc
         return $result === 0;
     }
 
+     /**
+     * Get usage stats of containers
+     * 
+     * @param string $container
+     * @param array<string, string> $filters
+     * 
+     * @return array
+     */
+    public function getStats(string $container = null, array $filters = []): array
+    {
+        // List ahead of time, since docker stats does not allow filtering
+        $containerIds = [];
+
+        $stdout = '';
+        $stderr = '';
+
+        if($container === null) {
+            if(\count($filters) > 0) {
+                $filterString = '';
+
+                foreach($filters as $key => $value) {
+                    $filterString = $filterString . ' --filter "'.$key.'='.$value.'"';
+                }
+
+                $result = Console::execute('docker ps --all --no-trunc --format "id={{.ID}}"'.$filterString, '', $stdout, $stderr);
+
+                if($result !== 0) {
+                    throw new Orchestration("Docker Error: {$stderr}");
+                }
+
+                $stdoutArray = \explode("\n", $stdout);
+
+                foreach($stdoutArray as $row) {
+                    if(empty($row)) {
+                        continue;
+                    }
+
+                    $rowParsed = [];
+                    \parse_str($row, $rowParsed);
+                    $containerIds[] = $rowParsed['id'];
+                }
+            }
```

Comment:
https://github.com/utopia-php/orchestration/blob/114046b9010ccb331deb7e23ef4aff22f903b890/src/Orchestration/Adapter/DockerCLI.php#L291

We can use the list function

Reply:
Used `list` instead.